layouts/page/cudf-pandas.html

```diff
@@ -162,6 +162,12 @@ <h2><i class="fa-light fa-bookmark"></i> Try Now on Colab</h2>
             your Google account by <a href="https://nvda.ws/3yW6j22" target="_blank">launching on Colab <i
                 class="fa-solid fa-arrow-up-right"></i> </a></p>
           <br>
+          <p>Prefer to use your own GPU? Get started with
+            <a href="https://github.com/rapidsai-community/showcase/blob/main/getting_started_tutorials/cudf_pandas_demo.ipynb"
+            target="_blank">10 Minutes to cudf.pandas</a> or check out our full collection of RAPIDS
+            <a href="https://github.com/rapidsai-community/notebooks-contrib/tree/main/getting_started_materials/intro_tutorials_and_guides"
+            target="_blank">community notebooks<i class="fa-solid fa-arrow-up-right"></i> </a> </p>
```

Comment:
Needs a space here for consistency.
```suggestion
            target="_blank">community notebooks <i class="fa-solid fa-arrow-up-right"></i> </a> </p>
```